assets/src/components/activities/multi_input/MultiInputDelivery.tsx

```diff
@@ -231,6 +239,10 @@ export const MultiInputComponent: React.FC = () => {
           deferredSaves.current[id].save(fn);
         }
       }
+
+      if (!isInputDirty[id]) {
```

Comment:
There is a case on line 234 that saves the part, this looks like it will set it to dirty anyways.

Probably not a big deal, but if we care, could move this up to the top and have handlePerPartSubmission clear the flag on save instead of it being after each call.